src/main/java/bio/terra/cli/command/helperclasses/FormatFlag.java

```diff
@@ -0,0 +1,101 @@
+package bio.terra.cli.command.helperclasses;
+
+import bio.terra.cli.command.exception.SystemException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import java.io.PrintStream;
+import java.util.function.Consumer;
+import picocli.CommandLine;
+
+/**
+ * Command helper class that defines the --format flag and provides utility methods to printing a
+ * return value out in different formats.
+ */
+public class FormatFlag {
+
+  @CommandLine.Option(
+      names = "--format",
+      hidden = true,
+      description = "Set the format for printing command output: ${COMPLETION-CANDIDATES}")
+  private FormatOptions format;
+
+  /** This enum specifies the format options for printing the command output. */
+  private enum FormatOptions {
+    json,
+    text;
+  }
+
+  // output stream to use for writing command return value
+  private static final PrintStream OUT = System.out;
+
+  /**
```

Comment:
Nit: should this be "calls printText method on the return value object" (instead of "of the return value object")?

Reply:
Agreed this was unclear. Changed to "calls the {@link #printText} method, passing the return value object as an argument."